01-data-structures/02-stacks-and-queues/myqueue/myqueue_spec.rb

```diff
@@ -6,26 +6,15 @@
   let(:q) { MyQueue.new }
 
   describe "#enqueue" do
-
-    it "updates the head and tail accessors properly" do
+    it "enqueues strings to the queue, verifies with head and tail" do
```

Comment:
Anything that says 2 things is bad. You should verify 1 thing per test. When you rename it, I can make sure you're only doing the thing it says to do